### PR TITLE
Add oracle price abstaining script

### PR DESCRIPTION
**Pull request by LCyson**
Tested locally with 1.0.4beta sei-chain: 
`python3 abstain_oracle_voting.py cyson sei-chain 12345678 ""`

### CODE REVIEW

File: scripts/oracle/abstain_oracle_voting.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+    print(args)
+
+    # fetch validator address if not provided
+    if not VAL_ADDR:
+        ADDR = subprocess.check_output([CMD_TMPL.format(password=PASSWORD) + f" keys show {KEY} -a"], stderr=subprocess.STDOUT,
+            shell=True,).decode()[:-1]
+
+        VAL_ADDR = subprocess.check_output(
+            [CMD_TMPL.format(password=PASSWORD) + f" query staking delegations {ADDR} | grep validator_address | cut -d':' -f2 | xargs"],
```
Comment (philipsu522) [inline scripts/oracle/abstain_oracle_voting.py]: this can just be queried by 
```
seid status | jq ".ValidatorInfo.Address"
```

Comment (philipsu522) [reply scripts/oracle/abstain_oracle_voting.py]: or `seid keys show <> --bech val`

File: scripts/oracle/abstain_oracle_voting.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    " tx oracle aggregate-prevote abc 0uusdc,0uatom {val_addr} --from={key} "
+    "--chain-id={chain_id} --fees=2000usei -y --broadcast-mode=sync"
+)
+VOTE_TMPL = (
+    " tx oracle aggregate-vote abc 0uusdc,0uatom {val_addr} --from={key} "
+    "--chain-id={chain_id} --fees=2000usei -y --broadcast-mode=sync"
+)
+
+KEY = "default_account"
+CHAIN_ID = "sei-internal-testnet"
```
Comment (philipsu522) [inline scripts/oracle/abstain_oracle_voting.py]: should we change this to sei-testnet-2 just so folks don't mess it up and ask questions

**Review by philipsu522 — APPROVED** [review]

File: scripts/oracle/abstain_oracle_voting.py (hunk 2 above)
Comment (LCyson) [reply scripts/oracle/abstain_oracle_voting.py]: will do

File: scripts/oracle/abstain_oracle_voting.py (hunk 1 above)
Comment (LCyson) [reply scripts/oracle/abstain_oracle_voting.py]: fixed

**Pull request merged**